app/views/NewOrUpdateCaseView.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        errorSummary: components.errorSummary,
+        customRadio: components.customYesNoRadio,
+        button: components.button)
+
+@(form: Form[_], backLink: Call)(implicit request: Request[_], messages: Messages)
+
+    @layout(
+        pageTitle = messages("newOrUpdate.title"),
+        form = Some(form),
+        customBackLinkUrl = Some(backLink)) {
+
+        @errorSummary(form.errors)
+
+        @formHelper(action = controllers.routes.NewOrUpdateCaseController.onSubmit()) {
+            @customRadio(
```

Comment:
Rather than create a new component you can use the existing inputRadio component and just add the hints to the Seq[RadioItems].